### PR TITLE
LPS-184962 Create new DDMFormFieldType RichTextLocalizable to be used on DDM Form.

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPS-184962](https://issues.liferay.com/browse/LPS-184962)
[LPS-184961](https://issues.liferay.com/browse/LPS-184961)

This pr depends on these : 
https://github.com/liferay-objects/liferay-portal/pull/2122
https://github.com/liferay-objects/liferay-portal/pull/2136

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:forms

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:forms

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableRichText/LocalizableRichText.es.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+					<LocalesDropdown
+						availableLocales={currentAvailableLocales}
+						editingLocale={currentEditingLocale}
+						fieldName={fieldName}
+						onLanguageClicked={({localeId}) => {
+							const newEditingLocale = currentAvailableLocales.find(
+								(availableLocale) =>
+									availableLocale.localeId === localeId
+							);
+							setCurrentEditingLocale({
```
Comment (naraandrade) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableRichText/LocalizableRichText.es.js:168]: @pedro-oliveira446 Please, add a blank line between the creation of a const and its usage.

**Review by naraandrade — CHANGES_REQUESTED** [review]
Hey @pedro-oliveira446, great job in this PR! I just asked some small format changes. Also, the values for the translated fields don't seem to be returned when we open an entry in the UI. Could you take a look at it, please? Thanks!

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:forms

**Comment by pedro-oliveira446** [comment]
Hi @naraandrade, thanks for the review,
the problem you found should be solved by downloading @selton's other pr related to the story as well. Sorry about that.

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by marcelabc** [comment]
Hey @guilhermedcamacho,
Could you help me to review this PR, please? Thank you!

**Comment by guilhermedcamacho** [comment]
No problem @marcelabc :wink:

**Review by marcelabc — CHANGES_REQUESTED** [review]
As I talked with @pedro-oliveira446, the backend solution was not ideal. We must refactor the solution in order to reuse the existing code from the Rich Text in DDM.

**Comment by marcelabc** [comment]
I'm closing this PR because the implementation will require refactoring.
Thank you @pedro-oliveira446 !
cc: @samuelromeiro @guilhermedcamacho

**Pull request closed without merging**